pkg/manifests/manifests_test.go

```diff
@@ -1046,6 +1052,92 @@ ingress:
 	}
 }
 
+func TestAdditionalAlertManagerConfigsSecret(t *testing.T) {
+	c, err := NewConfigFromString(`prometheusK8s:
+  additionalAlertManagerConfigs:
+  - apiVersion: v2
+    bearerToken:
+      name: alertmanager1-bearer-token
+      key: token
+    scheme: https
+    tlsConfig:
+      ca: 
+        name: alertmanager1-ca
+        key: root-ca.crt
+    pathPrefix: /
+    staticConfigs:
+    - alertmanager1-remote.com
+    - alertmanager1-remotex.com
+  - apiVersion: v2
+    scheme: https
+    timeout: 60s
+    tlsConfig:
+      ca: 
+        name: alertmanager2-cert
+        key: root-ca.crt
+      cert:
+        name: alertmanager2-cert
+        key: cert.crt
+      key:
+        name: alertmanager2-cert
```

Comment:
can you use a different secret name? it would detect the typo in `getAlertmanagerTLSSecretsName`.

Reply:
Here I would like to test the case that `ca` `cert` `key` are in the same secret. Let me add another case to have different secret for `ca` `cert` `key`